packages/components/src/components/hds/badge/index.ts

```diff
@@ -129,8 +133,8 @@ export default class HdsBadgeIndexComponent extends Component {
    * @method Badge#classNames
    * @return {string} The "class" attribute to apply to the component.
    */
-  get classNames() {
-    let classes = ['hds-badge'];
+  get classNames(): string {
```

Comment:
In other components (eg. the `Text` component) this is not defined as `string`, which one of the two approaches is "correct"?

https://github.com/hashicorp/design-system/blob/main/packages/components/src/components/hds/text/index.ts#L128

Reply:
Yea I mentally skipped over these. My bad. I've tested it out and typescript implicitly sets the return type. There is no harm in adding an explicit return type here in that it will only restrict accidentally changing the return type. 

<img width="464" alt="Screenshot 2024-03-26 at 9 51 41 AM" src="https://github.com/hashicorp/design-system/assets/5448834/bd40db45-1574-4aca-80ea-5559c052c7a4">

We don't have a guideline around this right now, but I'm going to move it to the way it's been done in the other components so far for consistency.  Happy to return to it after a discussion with Alex once he's back from PTO.